src/web/mjs/engine/Storage.mjs

```diff
@@ -779,18 +779,20 @@ export default class Storage {
      */
     sanatizePath(path) {
         if (this.platform.indexOf('win') === 0) {
-            // TODO: max. 260 characters per path
```

Comment:
The `TODO` is not resolved with this pull request, so it should be kept.
Other places as well in this file.

Reply:
Sure